### PR TITLE
[11.0] purchase_exception : Corrected runbot id

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: purchase_exception/README.rst [hunk 1]
```diff
@@ -39,6 +39,7 @@ Contributors
 ------------
 
 * Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (pedrobaeza) [inline purchase_exception/README.rst]: Please don't add you as contributor changing only the runbot ID.

**Review by anandkansagra — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
See my comment above

**Comment by JayVora-SerpentCS** [comment]
Updated

File: purchase_exception/README.rst (hunk 1 above)
Comment (JayVora-SerpentCS) [reply purchase_exception/README.rst]: Removed. I added contributor as a fact that we contributed to correct the wrong direction

**Review by bodedra — APPROVED** [review]
:+1:

**Comment by rousseldenis** [comment]
@OCA/procurement-purchase-maintainers We can merge this. Link is redirecting to runboat

**Pull request closed without merging**